README.md

```diff
@@ -28,94 +28,17 @@ Decidim is a participatory democracy framework written on Ruby on Rails original
 
```

Comment:
We have a severe case of badgetitis. On mobile is impossible to see the project's description without scrolling.  

![badges](https://cloud.githubusercontent.com/assets/717367/25193528/6e91d81e-2505-11e7-96bd-973c57fdc765.jpg)